supertokens_python/recipe/session/session_class.py

```diff
@@ -55,11 +62,9 @@ async def update_session_data(
 
     async def update_access_token_payload(
         self,
-        new_access_token_payload: Dict[str, Any],
-        user_context: Union[Dict[str, Any], None] = None,
+        new_access_token_payload: Union[Dict[str, Any], None],
+        user_context: Dict[str, Any],
```

Comment:
user_context should be optional here.. just like other functions in this class

Reply:
done.